students/14th/team_7/wd-shim/westagram/common_util/authorization.py

```diff
@@ -0,0 +1,38 @@
+import jwt
+import bcrypt
+
+from django.http import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from my_settings import SECRET, ALGO
+from user.models import User
+from user.const import UTF8
+
+def authorization_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            access_token = request.headers.get("Authorization", None)
+            payload      = jwt.decode(
+                access_token,
+                SECRET['secret'],
+                algorithms = ALGO
+            )
+            login_user   = User.objects.get(id=payload['id'])
+            request.user = login_user
+            
+        except jwt.exceptions.DecodeError:
+            return JsonResponse({"message": "INVALID_TOKEN"}, status=400)
+        
+        except ObjectDoesNotExist:
+            return JsonResponse({"message": "INVALID_USER"}, status=400)
```

Comment:
```suggestion
        except User.DoesNotExist:
            return JsonResponse({"message": "INVALID_USER"}, status=400)
```
어떤 클래스의 인스턴스가 없는지 명확히 지칭해주시면 더욱 정확한 에러 핸들링이 됩니다 :)

Reply:
ObjectDoesNotExist 삭제하고 일단 User.DoesNotExist 만 선언하였습니다.
프로젝트 빌드업 하면서 추가되는 발견되는 에러를 명확하게 선언하겠습니다.